docs(readme): add Hacktoberfest 2025 badge and update benchmarks

Akshay-datazip opened the pull request.

Akshay-datazip committed:
Update README.md

Signed-off-by: Akshay-datazip <akshay@datazip.io>

Akshay-datazip committed:
Update CONTRIBUTING.md

Signed-off-by: Akshay-datazip <akshay@datazip.io>

hash-data committed:
Merge pull request #538 from datazip-inc/staging

chore: releasing version 0.2.5

ImDoubD-datazip committed:
Merge branch 'master' into patch-2

ImDoubD-datazip committed:
Merge branch 'staging' into patch-1

Akshay-datazip committed:
Merge branch 'pr-541' into pr-540

Akshay-datazip committed:
Update Postgres benchmarks: 235,411 RPS (515.1× faster than Airbyte)

vaibhav-datazip reviewed (commented):
Also tags and labels for hacktoberfest needs to be added.

shubham19may commented on README.md:
```diff
@@ -50,14 +53,11 @@
 | Source → Destination | Throughput            | Relative Performance        |Full Report        |
 |----------------------|-----------------------|-----------------------------|-------------------|
-| Postgres → Iceberg   | 46,262 RPS (Full load)| 101× faster than Airbyte    |[Full Report](https://olake.io/docs/connectors/postgres/benchmarks) |
+| Postgres → Iceberg   | 2,35,411 RPS (Full load)| 515.1× faster than Airbyte    |[Full Report](https://olake.io/docs/connectors/postgres/benchmarks) |
```
we should compare ourselves with debezium instead airbyte.

Akshay-datazip committed:
Update CONTRIBUTING.md

Co-authored-by: Vaibhav <vaibhav@datazip.io>
Signed-off-by: Akshay-datazip <akshay@datazip.io>

Akshay-datazip committed:
docs: update benchmarks and refine Hacktoberfest contribution guidelines

- Update Postgres benchmarks to 14,839 RPS (15.9× faster than Debezium)
- Simplify contribution guidelines by removing 'help wanted' label
- Update benchmark report links and table formatting

vaibhav-datazip commented on README.md:
```diff
@@ -48,16 +51,13 @@
-| Oracle → Object Store (Parquet)     | WIP          |                             | |
+| Source → Destination | Throughput            | Relative Performance        |Full Report                            |
+|----------------------|-----------------------|-----------------------------|---------------------------------------|
+| Postgres → Iceberg   | 14,839 RPS (Full load)| 15.9× faster than Debezium (memiiso)|[Full Report](https://olake.io/docs/benchmarks?tab=postgres) |
```
14k rps in full load, in postgres ?

Akshay-datazip committed:
fix: correct Postgres benchmarks to 235,411 RPS

- Fix merge conflict that incorrectly showed 14,839 RPS
- Restore correct benchmark value of 235,411 RPS (Full load)
- Maintain 15.9× faster than Debezium comparison
